File: src/com/mapdigit/game/drop/actor/RainDrop.scala
package com.mapdigit.game.drop.actor

import com.guidebee.game.GameEngine._
import com.guidebee.game.graphics.Texture
import com.guidebee.game.scene.Actor

/**
 * Created by james on 5/10/2014.
 */
class RainDrop extends Actor("RainDrop"){
  setTexture(assetManager.get("droplet.png", classOf[Texture]))
}

File: src/com/mapdigit/game/tutorial/drop/actor/RainDrop.java
package com.mapdigit.game.tutorial.drop.actor;

import com.guidebee.game.GameEngine;
import com.guidebee.game.graphics.Pixmap;
import com.guidebee.game.graphics.SVGImage;
import com.guidebee.game.graphics.Texture;
import com.guidebee.game.graphics.TextureAtlas;
import com.guidebee.game.scene.Actor;
import com.guidebee.utils.Disposable;

import static com.guidebee.game.GameEngine.assetManager;

public class RainDrop extends Actor  {



    public RainDrop(){
        super("RainDrop");
        setTexture(assetManager.get("coin.png",Texture.class));

    }



}
